dashboard/src/components/ErrorAlert/ErrorAlert.tsx

```diff
@@ -45,8 +45,10 @@ export default function ErrorAlert({ error, children }: IErrorAlert) {
   }
   return (
     <Alert theme="danger">
-      {messages}
-      {children}
+      <>
+        {messages}
+        {children}
+      </>
```

Comment:
Is this the type related issue? Don't understand why we need an extra nesting here given it's already nested inside the `<Alert />`?

Reply:
I was getting this error:

````
This JSX tag's 'children' prop expects a single child of type 'string | number | boolean | ReactElementLike | ReactNodeArray', but multiple children were provided.  TS2746

    46 |   }
    47 |   return (
  > 48 |     <Alert theme="danger">
       |      ^
    49 |       {messages}
    50 |       {children}
    51 |     </Alert>
````

I don't know why the `PropTypes.node` type is not the same as the `React.ReactNode`... however, I've just changed the type definition to `children: React.ReactNode || PropTypes.node` and removed the highlighted workaround and it works now.